iphone/Classes/TiUIiPhoneTableViewScrollPositionProxy.m

```diff
@@ -17,10 +17,10 @@ -(NSString*)apiName
 }
 
 
-MAKE_SYSTEM_PROP(NONE,UITableViewScrollPositionNone);
-MAKE_SYSTEM_PROP(TOP,UITableViewScrollPositionTop);
-MAKE_SYSTEM_PROP(MIDDLE,UITableViewScrollPositionMiddle);
-MAKE_SYSTEM_PROP(BOTTOM,UITableViewScrollPositionBottom);
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(NONE,  UITableViewScrollPositionNone,@"UI.iPhone.ListViewScrollPosition.NONE",@"6.0.0",@"UI.iOS.ListViewScrollPosition.NONE");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(TOP,   UITableViewScrollPositionTop,@"UI.iPhone.ListViewScrollPosition.TOP",@"6.0.0",@"UI.iOS.ListViewScrollPosition.TOP");
```

Comment:
I did the indentation this way purposly so all the property namespaces line up with each other. It was like that for some of the old classes, also the new ones i added I've done the same. So I've tried to keep them the same where possible. For some of the deprecated classes/namespace i didn't do it as the deprecated message was too long and it made it impossible.